src/components/MovieDetails/MovieDetails.stories.tsx

```diff
@@ -1,4 +1,5 @@
 import { Meta, StoryObj } from "@storybook/react";
+import {fn} from "@storybook/test";
```

Comment:
when backend is not ready maybe we need to show NO FIlms?
![image](https://github.com/user-attachments/assets/063bbd0b-6172-4324-8df3-1f2d7bb64ed8)

Reply:
There is a message when there are no movies to display
![image](https://github.com/user-attachments/assets/7ac778a6-efd8-4340-b67d-581dcfa43d7f)